packages/gatsby-remark-prismjs/src/directives.js

```diff
@@ -109,8 +109,8 @@ module.exports = function highlightLineRange(code, highlights = []) {
   if (highlights.length > 0 || HIGHLIGHT_DIRECTIVE.test(code)) {
     // HACK split plain-text spans with line separators inside into multiple plain-text spans
     // separated by line separator - this fixes line highlighting behaviour for jsx
```

Comment:
Is this comment still accurate?

Reply:
Other than mentioning just `plain-text`, it is accurate, I will update it to mentioned "for example plain-text" and list couple of others token types.

Great catch! Thanks!